README.md

```diff
@@ -9,19 +9,60 @@ This directory contains the main configuration file and lists of vocabularies fo
 
 ## src/
 
-This directory contains the source files for the harvesters to generate the manifest file and citation and keywords files.
+This directory contains the source files for the harvesters. 
 
 ## resources/
 
-This directory contains the manifest file for the list of vocabularies (manifest.json). It is automatically generated and contains the details about the vocabulary and where to access its citation and keywords files.
+This directory contains the manifest file for the list of vocabularies (manifest.json). It is automatically generated and contains the details about the vocabulary and where to access its configuration file. The configuration file will then point to the keywords file. The format of a thesaurus entry in the manifest file is as follows:
```

Comment:
I don't think we wanted the manifest.json to be automatically generated. I think we just wanted this to point to the configuration file, which in turn could be used by the harvester.